### ISSUE TITLE
Plugin won't load

### ISSUE THREAD

**Issue by GonzoTG** [bigmoby/fglair_for_homeassistant#28]
I am trying to get this installed since I see it's not a dead repository (like so many others) and am getting this error:
I have reviewed the installation steps which seems to be authentication related. I am running this as a docker but did get to bash and verified that the required library **pyfujitseu** is the correct version. I'm still getting my legs under me with Home Assistant, so any nudges in the correct direction would be appreciated.

Logger: homeassistant.components.climate
Source: custom_components/fglair_heatpump_controller/climate.py:88 
Integration: Climate ([documentation](https://www.home-assistant.io/integrations/climate), [issues](https://github.com/home-assistant/home-assistant/issues?q=is%3Aissue+is%3Aopen+label%3A%22integration%3A+climate%22)) 
First occurred: 10:10:43 AM (1 occurrences) 
Last logged: 10:10:43 AM

Error while setting up fglair_heatpump_controller platform for climate
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 289, in _async_setup_platform
    await asyncio.shield(task)
  File "/usr/local/lib/python3.10/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/config/custom_components/fglair_heatpump_controller/climate.py", line 88, in setup_platform
    if not fglairapi._authenticate():
  File "/usr/local/lib/python3.10/site-packages/pyfujitseu/api.py", line 118, in _authenticate
    f = open(self._ACCESS_TOKEN_FILE, "w")
TypeError: expected str, bytes or os.PathLike object, not NoneType

Here is the configuration section:

```
climate:
   - platform: fglair_heatpump_controller
     username: !secret FGLAIR_USER
     password: !secret FGLAIR_PASS
     region: 'us'

and the secrets.

FGLAIR_USER: "<email address>"
FGLAIR_PASS: "<password containing lower, upper case, numbers and special characters>"```

**Comment by bigmoby** [comment]
@GonzoTG have you tried to declare `tokenpath` param?

**Comment by chiefcomm** [comment]
@GonzoTG as described by @bigmoby as a first step I recommend to add tokenpath to your configuration.

````
climate
  - platform: fglair_heatpump_controller
    username: !secret FGLAIR_USER
    password: !secret FGLAIR_PASS
    region: 'us'
    tokenpath: 'token.txt'

**Comment by GonzoTG** [comment]
Thanks! Yup. that eliminated the error, but now I don't see the device and don't have anything in the logs to indicate that there's a problem.

``` 
2023-01-26 18:37:04.461 WARNING (SyncWorker_4) [homeassistant.loader] We found a custom integration fglair_heatpump_controller which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
```
Which I know is standard, but does indicate to me that it's being loaded. I didn't mean for this to become a support thread. If there's a better place to get assistance, happy to move this to there.

**Comment by GonzoTG** [comment]
Ok, here's what I got. I've reviewed the logs and don't see any glaring errors. It did seem that line 21 is odd.

[home-assistant-fglair-debug.log](https://github.com/bigmoby/fglair_for_homeassistant/files/10517745/home-assistant-fglair-debug.log)

**Comment by bigmoby** [comment]
Hi @GonzoTG it appears to me ok. What's the problem? Have you found a climate device in `developer-tools/state` section?

**Comment by GonzoTG** [comment]
> Hi @GonzoTG it appears to me ok. What's the problem? Have you found a climate device in `developer-tools/state` section?

No, I haven't found any reference to the integration in developer tools...

**Comment by GonzoTG** [comment]
Digging a bit further and trying to think more broadly. I did come across this:
![2023-01-27_06-03-59](https://user-images.githubusercontent.com/18699503/215071511-ce7ed02c-f075-4a6a-bde4-61e795057719.png)

(I tried removing the ' just in case that could be a cause of the issue.).

Shouldn't there be a card in the "Integrations" tab for the device? I was able to add a card on the dashboard, so perhaps I've just wasted a bunch of your time. Really sorry about that.
